cloudapi/insights/proto/gen.go

```diff
@@ -0,0 +1,5 @@
+// Package proto contains the Protobuf definitions used
+// by the k6 Insights.
+package proto
+
+//go:generate sh gen.sh
```

Comment:
I would prefer if you just embed the whole thing in here and *not* use `find`.

It will be nice if this actually has good chances of working on a windows machine, withouhg needing fixes.

Reply:
Done in [369297e](https://github.com/grafana/k6/pull/3100/commits/369297e10e1cecea4516cf5a3a7809528c550ed2).